resources/lib/gui.py

```diff
@@ -473,6 +473,12 @@ def _save_shortcuts( self, weEnabledSystemDebug = False, weEnabledScriptDebug =
                 # Inform user menu couldn't be saved
                 xbmcgui.Dialog().ok( __addon__.getAddonInfo( "name" ), __language__( 32097 ), __language__( 32094 ) )                 
 
+    def _try_decode(self, text, encoding="utf-8"):
+        try:
+            return text.decode(encoding)
+        except:
+            return text
```

Comment:
it would probably make more sense to put this into a separate module instead of adding it to gui class?